examples/tutorials/rigid_object_tutorial.py

```diff
@@ -1,44 +1,83 @@
-# [setup]
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.4.2
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %%
+# !curl -L https://raw.githubusercontent.com/facebookresearch/habitat-sim/master/examples/colab_utils/colab_install.sh | bash -s
+
+
+# %%
+# %run -i /content/habitat-sim/examples/colab_utils/load_env.py
+
+# %%
+# %cd /content/habitat-sim
 import math
-import os
-import random
+import os, sys
 
 import cv2
-import magnum as mn
+import imageio
+try:
+  import magnum as mn
+except ImportError:
+  if "google.colab" in sys.modules:
+    conda_path = '/usr/local/lib/python3.6/site-packages/'
+    user_path = '/root/.local/lib/python3.6/site-packages/'
```

Comment:
Can this environment-setting and path-mod code be put in a method in viz_utils.py so that it doesn't have to be repeated in every colab-friendly tutorial?  

Also, can these paths be put into some kind of global colab config, so that if/when we change to python 3.7 we don't have to go hunting for things to fix?

Reply:
It's not us who changes, they are values in Collab.

Reply:
Also easier said then done since using viz_utils would require habitat_sim to be properly be in the sys.path to begin with, right?

Reply:
I actually just figured out a way to remove the need to for this pesky code snippit entirely with some clever symlinking.